src/contribute-to-tidb/cherrypick-a-pr.md

```diff
@@ -0,0 +1,30 @@
+# Cherry-pick a Pull Request
+
+TiDB uses [release train model](../project-management/release-train-model.md) and has multiple releases. Each release matches one git branch. For `type/bug` issues with `severity/critical` and `severity/major`, it is anticipated to be fixed on any [currently maintained releases](https://pingcap.github.io/tidb-dev-guide/project-management/release-train-model.html#what-is-tidb-version-mechanism-under-release-train-model) if affected. Contributors and reviewers are responsible to settle the affected versions once the bug is identified as `severity/critical` or `severity/major`.
```

Comment:
I think we can have a subtitle:

What Kind of PRs are Good for Cherry Picks or What Kind of PRs need to Cherry Picks

like: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/cherry-picks.md#what-kind-of-prs-are-good-for-cherry-picks

Reply:
Definitely a good idea. But our cherry-pick strategy is not same as k8s. Currently `type/bug` issues with `severity/critical` and `severity/major` are cherry-picked. The QA team or Devs are responsible to diagnose and add the correct `severity/xx` but I don't know the principle. 

And I don't know what if we want to cherry-pick a low severity bug? Is it allowed?

Reply:
But for critical or major bugs, we need to be cherry-picked, this rule is clear to us, right? If there are more detailed and clear rules in the future, we can supplement them in this chapter.

Reply:
I add the bug severity identification guideline from PingCAP QA team. This is the guideline now to guide everyone diagnosing the severity of bug issues. Since severity is highly connected with cherry-pick, I think it's also the guideline of "What Kind of PRs are Good for Cherry Picks".